### DISCUSSION TITLE
Citation.cff

### DISCUSSION THREAD

**Discussion by rameengithub** [community/community#29744] [Other Feature Feedback, Questions, & Ideas]
How to delete a citation.cff file if I enter wrong DOI?

**Comment by airtower-luna** [comment]
The same way as any other file. On the command line use `git rm FILE` and then commit, on the web UI open the file and use the trash can icon on the right. :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/5814919/185051431-1f605b39-d059-4294-af39-c572445e869f.png)

Though it might be better to fix the citation than to delete the whole file. :wink:
